agent/bench-scripts/pbench-user-benchmark

```diff
@@ -30,209 +31,324 @@ benchmark="pbench-user-benchmark"
 # post-processing of benchmark specific data is the responibility of the user supplied benchmark
 
 # Defaults
-config=""
+export config=""
 tool_group=default
 sysinfo=default
 use_tool_triggers=0
 capture_stderr=" 2>&1"
+pbench_pre=""
+pbench_post=""
 
 function usage {
 	printf "Usage: $script_name [options] -- <script to run>\n\n"
 	printf "\tThe following options are available:\n\n"
-	printf -- "\t\t-C str --config=str            name of the test config\n"
-	printf -- "\t\t       --tool-group=str\n"
-	printf -- "\t\t       --sysinfo=str,          str = comma separated values of system information to be collected\n"
-	printf -- "\t\t                                     available: $(pbench-collect-sysinfo --options)\n"
-	printf -- "\t\t       --pbench-pre=str        path to the script which will be executed before tools are started\n"
-	printf -- "\t\t       --pbench-post=str       path to the script which will be executed after tools are stopped and postprocessing is complete\n"
-	printf -- "\t\t       --use-tool-triggers     use tool triggers instead of normal start/stop around script\n"
-	printf -- "\t\t       --no-stderr-capture     do not capture stderr of the script to the result.txt file\n"
+	printf -- "\t\t-C str --config=str   name of the test config\n"
+	printf -- "\t\t--tool-group=str      The tool group to use for the list of tools\n"
+        printf -- "\t\t--iteration-list=str  A list of iterations to run for the provided script;\n"
```

Comment:
"A file containing a list of iterations to run ..."

Reply:
Fixed.